# HG changeset patch
# User Enno Boland (tox) <tox@s01.de>
# Date 1262612380 -3600
# Node ID 620201658a0f2ece64615db8f8e1bd90623d2775
# Parent  150f5c9b19dc6362a3fb75610fca1d362d863f2d
dl has to be a directory, not a file.

--- a/config.def.h  Tue Dec 15 17:16:01 2009 +0100
+++ b/config.def.h  Mon Jan 04 14:39:40 2010 +0100
@@ -5,7 +5,7 @@
 static char *stylefile      = ".surf/style.css";
 static char *scriptfile     = ".surf/script.js";
 static char *cookiefile     = ".surf/cookies.txt";
-static char *dldir          = ".surf/dl";
+static char *dldir          = ".surf/dl/";
 static time_t sessiontime   = 3600;
 
 #define SETPROP(p)       { .v = (char *[]){ "/bin/sh", "-c", \
